test/core/test_load_builtin.py

```diff
@@ -0,0 +1,42 @@
+import importlib
+
+from mathics.core.load_builtin import (
+    _builtins,
+    add_builtins_from_builtin_module,
+    import_and_load_builtins,
+)
+from mathics.session import MathicsSession
+
+
+def test_add_builtins_from_builtin_module():
+    """
+    Test that add_builtins_from_module() loads a single Builtin module
+    and updates definitions.
+    """
+    # Set up a session with all but one module.
+    # Then evaluate a builtin in that module and see that we
+    # now have the function defined.
+
+    # First, load in many modules except quantum_mechanics.
+    _builtins = {}
+    import_and_load_builtins(exclude_files={"quantum_mechanics"}, clean_all=True)
```

Comment:
Pytest loads all the tests before running them. So, as other unit tests load  `test.helper`, and it calls `import_and_load_builtins`, when we reach this test, all the modules are already loaded. So, if we want to do a test over a "fresh" instance of the module, we need this second parameter.